libs/sdk-ui-tests-e2e/cypress/tools/widget.ts

```diff
@@ -144,4 +145,49 @@ export class Widget {
         this.getElement().should("have.class", "is-selected");
         return this;
     }
+
+    dragBefore(name: InsightTitle) {
+        return this.drag(name, "prev");
+    }
+
+    dragAfter(name: InsightTitle) {
+        return this.drag(name, "next");
+    }
+
+    drag(name: InsightTitle, offset: "next" | "prev") {
+        const catalog = new InsightsCatalog();
+        catalog.searchExistingInsight(name);
+        cy.get(catalog.getInsightSelector(name)).parent().trigger("dragstart", { force: true });
+        cy.wait(1000);
+        cy.get(`.dropzone.${offset}`).eq(this.index).trigger("dragenter");
+        cy.wait(1000);
+        return this;
+    }
+
+    add(name: InsightTitle, offset: "next" | "prev") {
+        const catalog = new InsightsCatalog();
+        catalog.searchExistingInsight(name);
+        cy.get(catalog.getInsightSelector(name)).parent().trigger("dragstart", { force: true });
+        cy.wait(1000);
```

Comment:
comment as above, try to delete wait then run again, if not passed then please reduce to 300ms, 1s is too long

Reply:
ok, I removed now, if it'll fail later, we'll re-add wait here